refarm_pagination/views.py

```diff
@@ -0,0 +1,30 @@
+from refarm_pagination.pagination import Paginator, NeighborPages
+
+
+class PaginationContext:
+
+    def __init__(self, url, number, per_page, objects):
+        self.url = url
+        self.number = number
+        self.paginator = Paginator(objects, per_page)
+
+    def context(self):
+        page = self.paginator.page(self.number)
+        showed_number = (self.number - 1) * self.paginator.per_page + len(page)
```

Comment:
`showed_count` name would be more clear here.
When i look on `showed_number`, i think about one number from rendered sequence `1 2 3 4`